daytrader3-ee6/build.gradle

```diff
@@ -95,34 +92,29 @@ String getHttpPort() {
     return DEFAULT_HTTP_PORT
 }
 
-libertyStart.mustRunAfter publishEar
+libertyStart.mustRunAfter 'publishEar'
 
 liberty {
     installDir = owner.hasProperty('libertyRoot') ? "${libertyRoot}" : null
-    serverName = 'daytrader3_Sample'
-    userDir = '../daytrader3-ee6-wlpcfg'
-    clean = true
-    install {
-       
+    server{
+        name = 'daytrader3_Sample'
+        userDir = '../daytrader3-ee6-wlpcfg'
+        looseApplication = false
+        features {
+           name = ['daytrader3_Sample']
+           acceptLicense = true
+        }

```

Comment:
instead of the copyEar task we could have 
apps = [ear.??? something to get the archive file]